fudge-ratio/src/main/webapp/about/about.js

```diff
@@ -1,30 +1,29 @@
 let currentSlide = 0;
 
 const openTab = tabName => {
-  Array.from(document.getElementsByClassName("bioContainer"))
-    .forEach(bio => bio.style.display = bio.id === tabName? "block": "none");
-}
-const closeTab = () => {
-  Array.from(document.getElementsByClassName("bioContainer"))
-    .forEach(bio => bio.style.display = "none");
-}
+  const elements = Array.from(document.getElementsByClassName("bioContainer"));
```

Comment:
We do not change the display prop this time, but the height and the padding, so the animation works

Reply:
Oh this is a cool trick!